client/app/project/project.html

```diff
@@ -371,6 +371,10 @@ <h3 style="display: inline-block;">{{ 'Mapping' | translate }}</h3>
                                         <!-- task action buttons -->
                                         <div class="form__group form__group--section">
                                             <div>
+                                                <div ng-show="projectCtrl.showWarning">
+                                                    <p class="error"><a class='emailWarning' href="/user/{{projectCtrl.user.username}}">{{ 'Add Your Email' | translate }}</a>
+                                                        {{ ' To Continue Mapping' | translate }}</p>
```

Comment:
I suggest to use `{{ 'to continue validation' }}` and put the white space before the translatable string. I also think we can go with normal not capitalized words. "Add your email address" "to continue validation".